docs/no_peeking/mappings/individual.py

```diff
@@ -0,0 +1,114 @@
+            'code': 'XY',
+            'display': 'Male'
+        }
+    elif x == constants.COMMON.OTHER:
+        return {
+            'system': 'http://ga4gh.org/fhir/phenopackets/CodeSystem/KaryotypicSex',
+            'code': 'OTHER_KARYOTYPE',
+            'display': 'None of the above types'
+        }
+    elif x == constants.COMMON.UNKNOWN:
+        return {
+            'system': 'http://ga4gh.org/fhir/phenopackets/CodeSystem/KaryotypicSex',
+            'code': 'UNKNOWN_KARYOTYPE',
+            'display': 'Untyped or inconclusive karyotyping'
+        }
+    else:
+        raise Exception('Unknown Individual KaryotypicSex')
+
+def taxonomy(x):
+    """
+    http://purl.obolibrary.org/obo/ncbitaxon.owl
+    """
+    if x == constants.SPECIES.DOG:
+        return {
+            'system': 'http://purl.obolibrary.org/obo/ncbitaxon.owl',
+            'code': 'NCBITaxon:9615',
+            'display': 'Canis lupus familiaris'
+        }
+    elif x == constants.SPECIES.HUMAN:
+        return {
+            'system': 'http://purl.obolibrary.org/obo/ncbitaxon.owl',
+            'code': 'NCBITaxon:9606',
+            'display': 'Homo sapiens'
+        }
+    else: 
+        raise Exception('Unknown Individual Taxonomy')
+
+def gender(x):
+    """
+    http://hl7.org/fhir/R4/valueset-administrative-gender.html
+    """
+    if x == constants.GENDER.FEMALE:
+        return 'female'
+    elif x == constants.GENDER.MALE:
+        return 'male'
+    elif x == constants.COMMON.OTHER:
+        return 'other'
+    elif x == constants.COMMON.UNKNOWN:
+        return 'unknown'
+    else:
+        raise Exception('Unknown Individual gender')
+
+
+individual = {
+    'resourceType': 'Patient',
+    'id': CONCEPT.PARTICIPANT.TARGET_SERVICE_ID,
+    'meta': {
+        'profile': ['http://ga4gh.fhir.phenopackets/StructureDefinition/Individual']
+    },
+    'extension': [
+        {
+            'text': 'KaryotypicSex',
+            'url': 'http://ga4gh.org/fhir/phenopackets/StructureDefinition/KaryotypicSex',
+            'valueCodeableConcept': {
+                'coding': [
+                    karyotypic_sex(CONCEPT.PARTICIPANT.GENDER)
+                ],
+                'text': CONCEPT.PARTICIPANT.GENDER
+            }
+        },
+        {
+            'text': 'Taxonomy',
+            'url': 'http://ga4gh.org/fhir/phenopackets/StructureDefinition/Taxonomy',
+            'valueCodeableConcept': {
+                'coding': [
+                    taxonomy(CONCEPT.PARTICIPANT.SPECIES or constants.SPECIES.HUMAN)
+                ],
+                'text': CONCEPT.PARTICIPANT.SPECIES or constants.SPECIES.HUMAN
+            }
+        }
+    ],
+    'identifier': [
+        {
+            'system': 'https://kf-api-dataservice.kidsfirstdrc.org/participants',
+            'value': CONCEPT.PARTICIPANT.TARGET_SERVICE_ID
+        },
+        {
+            'value': CONCEPT.PARTICIPANT.ID
```

Comment:
This could use a 'system' as well

Reply:
@fiendish will create a PR to this branch which adds a system attribute to the second set of IDs.